Android/Guava/source/Xamarin.CheckerFramework.CheckerCompatQual/Xamarin.CheckerFramework.CheckerCompatQual.targets

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <EmbeddedReferenceJar Include="$(MSBuildThisFileDirectory)..\..\jar\*.jar" />
```

Comment:
Trying to understand what this is doing here. Customers would add a `Xamarin.CheckerFramework.CheckerCompatQual` package and this is a regular binding library?

Should this use `AndroidJavaLibrary` with a condition instead? Like AndroidX does?

https://github.com/xamarin/AndroidX/blob/47d87a33df02bc49e753c74a26e5d26bc6556b2f/templates/kotlinx/Targets.cshtml#L6-L9

Reply:
Those libraries are annotation libraries, so no need for actual bindings.
I need nuget to satisfy dependencies in binderator step (AX, GPS-FB-MLKit)

Reply:
Right, I think `AndroidJavaLibrary` is still appropriate here because if you added this to a Xamarin.Android app project, `EmbeddedReferenceJar` wouldn't do anything.

Reply:
Yes, `EmbeddedReferenceJar` doesn't do anything in a Xamarin.Android application or class library project -- only binding projects?